pkg/processor/build/runtime/dotnetcore/docker/onbuild/handler.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
update 
https://github.com/nuclio/nuclio-sdk-dotnetcore/blob/master/nuclio-sdk-dotnetcore.csproj as well
and https://github.com/nuclio/nuclio/blob/268bd6b08e48bf3ce9f2738dd3311f182052dd97/pkg/processor/build/runtime/dotnetcore/docker/onbuild/Dockerfile#L36
and https://github.com/nuclio/nuclio/blob/268bd6b08e48bf3ce9f2738dd3311f182052dd97/pkg/processor/build/runtime/dotnetcore/docker/onbuild/Dockerfile#L22C2-L22C2